src/components/App/App.js

```diff
@@ -30,67 +29,47 @@ import {
 } from 'pages';
 import SuspenseFallback from 'components/SuspenseFallback/SuspenseFallback';
 
-export class App extends Component {
-  static propTypes = {
-    actions: PropTypes.object.isRequired
-  };
+export default function App() {
+  const dispatch = useDispatch();
 
-  componentDidMount() {
-    const { actions } = this.props;
+  useEffect(() => {
+    dispatch(appActions.initApp());
+  }, [dispatch]);
 
-    actions.initApp();
-  }
-
-  render() {
-    return (
-      <Suspense fallback={<SuspenseFallback />}>
-        <ErrorBoundary>
-          <Helmet defaultTitle="MixNJuice" titleTemplate="MixNJuice - %s" />
-          <Header />
-          <ToastDrawer />
-          <Switch>
-            <Route exact path="/" component={Home} />
-            <Route exact path="/login" component={Login} />
-            <Route exact path="/register" component={Register} />
-            <Route exact path="/recipes" component={Recipes} />
-            <PrivateRoute
-              exact
-              path="/recipe/editor"
-              component={RecipeEditor}
-            />
-            <Route exact path="/flavors" component={Flavors} />
-            <Route exact path="/recipe" component={Recipe} />
-            <PrivateRoute exact path="/user/profile" component={Profile} />
-            <PrivateRoute exact path="/user/recipes" component={UserRecipes} />
-            <PrivateRoute exact path="/user/favorites" component={Favorites} />
-            <PrivateRoute
-              exact
-              path="/user/flavor-stash"
-              component={FlavorStash}
-            />
-            <PrivateRoute
-              exact
-              path="/user/shopping-list"
-              component={ShoppingList}
-            />
-            <PrivateRoute
-              exact
-              path="/user/settings"
-              component={UserSettings}
-            />
-            <PrivateRoute exact path="/dashboard" component={Dashboard} />
-            <PrivateRoute path="/user/:userName" component={Profile} />
-            <Route component={NotFound} />
-          </Switch>
-          <Footer />
-        </ErrorBoundary>
-      </Suspense>
-    );
-  }
+  return (
+    <Suspense fallback={<SuspenseFallback />}>
+      <ErrorBoundary FallbackComponent={ErrorFallback}>
```

Comment:
This uses the new react-error-boundary component to render the error boundary in a functional way.